Makefile

```diff
@@ -347,6 +350,44 @@ bundle-run: operator-sdk ## Run bundle image
 bundle-cleanup: operator-sdk ## Remove bundle installed via bundle-run
 	$(OPERATOR_SDK) -n $(BUNDLE_RUN_NAMESPACE) cleanup $(OPERATOR_NAME)
 
+# Build a file-based catalog image
+# https://docs.openshift.com/container-platform/4.14/operators/admin/olm-managing-custom-catalogs.html#olm-managing-custom-catalogs-fb
+# NOTE: CATALOG_DIR and CATALOG_DOCKERFILE items won't be deleted in case of recipe's failure
+CATALOG_DIR := catalog_dir
+CATALOG_DOCKERFILE := ${CATALOG_DIR}.Dockerfile
+CATALOG_INDEX := $(CATALOG_DIR)/index.yaml
+
+.PHONY: catalog-build
+catalog-build: opm ## Build a file-based catalog image.
+	# Remove the catalog directory and Dockerfile if they exist
+	-rm -r ${CATALOG_DIR} ${CATALOG_DOCKERFILE}
+	@mkdir -p ${CATALOG_DIR}
+	$(OPM) generate dockerfile ${CATALOG_DIR}
+	$(OPM) init ${OPERATOR_NAME} \
+		--default-channel=${CHANNELS} \
+		--description=./README.md \
```

Comment:
Isn't that too long description?

Reply:
I actually didn't look into the resulting catalog's description, but I followed what is suggested at https://docs.openshift.com/container-platform/4.14/operators/admin/olm-managing-custom-catalogs.html step C (or at [opm CLI reference - init](https://docs.openshift.com/container-platform/4.15/cli_reference/opm/cli-opm-ref.html#opm-cli-ref-init_cli-opm-ref))
Since we don't have a shorter documentation file, I think this one could be fine right now.

Reply:
Since I don't know how the index image would like with this (long) description, so I would leave it up to you.
BTW we have a short description in the CSV that we use https://github.com/medik8s/machine-deletion-remediation/blob/main/config/manifests/bases/machine-deletion-remediation.clusterserviceversion.yaml#L10

Reply:
> BTW we have a short description in the CSV that we use

we still need to provide a _filepath_ here